examples/content.html

```diff
@@ -1,156 +1,413 @@
 ---
 altLangPage: "content-fr.html"
-dateModified: "2025-04-08"
+dateModified: "2025-05-15"
 description: "Office of the Auditor General of Canada content page"
 language: "en"
 pageclass: "oag-theme wb-prettify all-pre"
-title: "OAG - content page"
+title: "Content page"
 layout: default
 ---
 
-<p>Note: This page showcases examples of CSS classes, components, and other elements from GCWeb that have been customized or overwritten for this project. These modifications were made to better align with the specific requirements and design goals of the project.</p>
+<div class="alert alert-info">
+	<h2 class="h4">Note</h2>
+	<p>This page showcases examples of CSS classes, components, and other elements from GCWeb that have been customized or overwritten for this project. These modifications were made to better align with the specific requirements and design goals of the project.</p>
```

Comment:
No heading required, that impact the structure for the following paragraph outside the alert scope

```suggestion
	<p><strong>Note:</strong> This page showcases examples of CSS classes, components, and other elements from GCWeb that have been customized or overwritten for this project. These modifications were made to better align with the specific requirements and design goals of the project.</p>
```

If you want to add an heading, then we need to use a section tag.